index.html

```diff
@@ -50,6 +50,92 @@ <h2 class="iam">I'm Sotfware Developer</h2>
+        been the industry's
+        standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+      </p>g
+      <ul class="tag-recent-work">
+        <li> CSS</li>
+        <li>HTML</li>
+        <li>Bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <button type="button" class="button-see-project">See project</button>
+    </div>
+    <div class="container-works grid">
+      <div class="card-content">
+        <h3 class="work-title">Profesional Art Printing Data</h3>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+        </p>
+        <ul class="tag">
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="see-project-card">See Project</button>
+      <div class="card-content">
+        <h3 class="work-title">Profesional Art Printing Data</h3>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+        </p>
+        <ul class="tag">
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="see-project-card">See Project</button>
+      <div class="card-content">
+        <h3 class="work-title">Profesional Art Printing Data</h3>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+        </p>
+        <ul class="tag">
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="see-project-card">See Project</button>
+      <div class="card-content">
+        <h3 class="work-title">Profesional Art Printing Data</h3>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+        </p>
+        <ul class="tag">
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="see-project-card">See Project</button>
+      <div class="card-content">
+        <h3 class="work-title">Profesional Art Printing Data</h3>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+        </p>
+        <ul class="tag">
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="see-project-card">See Project</button>
+    </div>
+  </section>
```

Comment:
- [ ] Kindly add the about me, languages, skills and frameworks section after the works section 👍🏿 

![Screenshot_53](https://user-images.githubusercontent.com/69638013/164956470-a23ff606-3c5c-47f5-8b16-9f167a666764.png)

Reply:
| First Header  
![Screenshot from 2022-04-30 09-28-32](https://user-images.githubusercontent.com/77842237/166094604-90fbf4dd-51de-4e3c-b3c0-ff9c41a78e37.png)


![Screenshot from 2022-04-30 09-21-40](https://user-images.githubusercontent.com/77842237/166094464-42471a3f-8c65-4d99-95a8-19c534e4d296.png)

- [ ] [REQUESTED] Kindly as requested by the previous reviewer you must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).